index.html

```diff
@@ -8,34 +8,204 @@
-            <p>Hello I’m a software developer! I can help you build a product , feature or website
-                 Look through some of my work and experience! If you like what you see and have a 
-                 project you need coded, don’t hestiate to contact me.</p>
+            <div class="container">
+                <nav class="nav_bar">
+                        <span class="myLogo"><a href="https://user-images.githubusercontent.com/34779928/205124171-ac56355c-3654-48b2-98c3-b053b661f64d.png"><img src="image/My_Logo.svg" alt="My Logo"></a></span>
+                        <span class="contact"><a href="https://user-images.githubusercontent.com/34779928/205124171-ac56355c-3654-48b2-98c3-b053b661f64d.png"><img src="image/Icon - Menu.svg" alt="Contact Us"></a></span>
+                </nav>
+            <div class="img_background">
+                    <h1 class="header" >I’m Mariane<br> 
+                        Glad to see<br> you!</h1>
+                    <p class="content">Hello I’m a software developer! I can help you build a product , feature or website
+                         Look through some of my work and experience! If you like what you see and have a 
+                         project you need coded, don’t hestiate to contact me.</p>
+                <div class="connect"><a href="https://user-images.githubusercontent.com/34779928/205124171-ac56355c-3654-48b2-98c3-b053b661f64d.png"><img src="image/Social Media.svg" alt="Let's Connect"></a></div>
+                 <div class="social">
+                    <ul>
+                        <li><a href="https://user-images.githubusercontent.com/34779928/205124171-ac56355c-3654-48b2-98c3-b053b661f64d.png"><img class="twitter" src="image/twitter.svg" alt="twitter"></a></li>
+                        <li><a href="https://user-images.githubusercontent.com/34779928/205124171-ac56355c-3654-48b2-98c3-b053b661f64d.png"><img class="linkedin" src="image/Linkedin icon.svg" alt="Linkedin"></a></li>
+                        <li><a href="https://user-images.githubusercontent.com/34779928/205124171-ac56355c-3654-48b2-98c3-b053b661f64d.png"><img class="medium" src="image/medium.svg" alt="Medium"></a></li>
+                        <li><a href="https://user-images.githubusercontent.com/34779928/205124171-ac56355c-3654-48b2-98c3-b053b661f64d.png"><img class="github" src="image/github.svg" alt="GitHub"></a></li>
+                        <li><a href="https://user-images.githubusercontent.com/34779928/205124171-ac56355c-3654-48b2-98c3-b053b661f64d.png"><img class="angellist" src="image/angellist.svg" alt="Angellist"></a></li>
+                    </ul>
+                 </div>
+            </div>
+            </div>
         </section>
+
+        <section>
+            <div class="border_project2">
+                <div class="image_project2">
+                    <img src="image/Snapshoot Portfolio.svg" alt="Snapshoot Portfolio">
+                </div>
+                <h1 class="tonic">Tonic</h1>
```

Comment:
- [ ] Kindly please change the h1 tag to some other heading tag from (h2-h6) in one page needs only one h1. It's a convention and accessibility reasons. For further details [check this](https://www.socialhive.com.au/blog/2021/june/h1-tags/#:~:text=HTML%20heading%20tags%20are%20used,language%20used%20to%20create%20websites.)